src/scripts/main.js

```diff
@@ -1,3 +1,171 @@
 'use strict';
 
-// write code here
+const form = document.createElement('form');
+let tbody = document.querySelector('tbody');
+const thead = document.querySelector('thead');
+const collectionOfButton = [...thead.rows[0].cells];
+
+form.className = 'new-employee-form';
+
+// add form
+form.insertAdjacentHTML(
+  'afterbegin',
+  `
+  <label>Name: <input  required  name="name" data-qa="name"  type="text"></label>
+  <label>Position: <input required name="position" data-qa="position"  type="text"></label>
+  <label>Office: <select data-qa="office" required>
+  <option value="Tokyo">Tokyo</option>
+  <option value="Singapore">Singapore</option>
+  <option value="London">London</option>
+  <option value="New York">New York</option>
+  <option value="Edinburgh">Edinburgh</option>
+  <option value="San Francisco">San Francisco</option>
+  </select> </label>
+  <label>Age: <input required name="age" data-qa="age"  type="number"></label>
+  <label>Salary: <input required name="salary" data-qa="salary"  type="number"></label>
+  <button type="submit">Save to table</button>`,
+);
+document.body.append(form);
+
+// add notification
+function showNotification(message, type) {
+  const notification = document.createElement('div');
+
+  notification.classList.add('notification');
+  notification.classList.add(type);
+  notification.setAttribute('data-qa', 'notification');
+
+  notification.textContent = message;
+
+  document.body.append(notification);
+
+  setTimeout(() => {
+    notification.remove();
+  }, 2000);
+}
+
+// sort the table
+let isAscending = true;
+
+function sortTableBySalary(isAscending) {
```

Comment:
Try to fix this lint error, but it might be difficult since all logic is in one file. 
![image](https://github.com/user-attachments/assets/d7a9115c-89d5-42d9-add9-fa385f0c9c86)

So at least disable this lint rule for the entire file. How to do it:
https://stackoverflow.com/questions/34764287/turning-off-eslint-rule-for-a-specific-file